core-common/src/main/java/org/apache/kylin/common/persistence/JDBCResourceStore.java

```diff
@@ -44,28 +45,17 @@
 
 public class JDBCResourceStore extends PushdownResourceStore {
 
-    private static Logger logger = LoggerFactory.getLogger(JDBCResourceStore.class);
-
     public static final String JDBC_SCHEME = "jdbc";
-
+    private static final ConcurrentHashMap<String, Object> resourceHashMap = new ConcurrentHashMap<>();
```

Comment:
Maybe change the name `resourceHashMap` to something like `lockObjectMap` is more readable.

Reply:
@hit-lacus Good idea